### PR TITLE
Implemented the JWKS endpoint at /oauth2/connect/keys

### LINKED ISSUE #8: Add HTTP endpoint to list all known public keys
We should add an endpoint to list all known public keys in order to verify that the tokeninfo service is up-to-date.

### PR DESCRIPTION
**Pull request by lmineiro**
This closes #8

### CODE REVIEW

File: handlers/jwks/handler.go [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+// ServeHTTP serializes the current snapshot of Keys from the KeyLoader as a JSON Web Key Set
+func (h *jwksHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
+	wrapper := &jwksWrapper{keys: h.loader.Keys()}
+	w.Header().Set("Content-Type", "application/json")
+	if err := json.NewEncoder(w).Encode(wrapper); err != nil {
+		w.WriteHeader(http.StatusInternalServerError)
+		log.Println("Failed to finish JWKS response: ", err)
+	} else {
+		w.WriteHeader(http.StatusOK)
```
Comment (aryszka) [inline handlers/jwks/handler.go]: not, sure, i need to look up the what the Encoder.Encode does, but i would bet that it is already too late here to set the status

Comment (lmineiro) [inline handlers/jwks/handler.go]: Check the tests for it

**Comment by hjacobs** [comment]
:+1:

File: handlers/jwks/wrapper.go [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+		m, err := fromJwk(key)
+		if err != nil {
+			fmt.Printf("Failed to convert the JWK %v to a map: %v\n", key, err)
+			return nil, err
+		}
+		keys[i] = m
+		i++
+	}
+
+	m["keys"] = keys
```
Comment (aryszka) [inline handlers/jwks/wrapper.go]: this map is a bit confusing with the 'm' var in the loop, and triggers circular structure alert in the reader. can we remove the outer 'm' var, and just call below simply:

```
return json.Marshal(map[string][]map[string]string{"keys": keys})
```

?

**Comment by aryszka** [comment]
lgtm

**Comment by aryszka** [comment]
even better

**Pull request merged**